MapboxNavigation/NavigationMapView.swift

```diff
@@ -22,6 +22,24 @@ let sourceOptions: [MGLShapeSourceOption: Any] = [.maximumZoomLevel: 16]
 @objc(MBNavigationMapView)
 open class NavigationMapView: MGLMapView {
     
+    //MARK: Class Constants
+    
+    /**
+     `defaultAltitude` is the altitude that the map camera initally defaults to.
```

Comment:
Nit: no need to repeat the name of the property inside its own documentation. Sometimes we do that for classes, but I guess that’s because we try to be less terse there.